samcli/commands/validate/lib/sam_template_validator.py

```diff
@@ -180,3 +208,11 @@ def _update_to_s3_uri(property_key, resource_property_dict, s3_uri_value="s3://b
             return
 
         resource_property_dict[property_key] = s3_uri_value
+
+    @staticmethod
+    def _replace_with_file_contents(resource_property_dict, location_prop):
+        definition_body = parse_yaml_file(location_prop)
+
+        # replace the definition body from the file
+        resource_property_dict["DefinitionBody"] = definition_body
+        LOG.debug("Imported definition body from %s", location_prop)
```

Comment:
Nit: Can you move this above line 217. This doesn't have meaningful impact but I find it to be more consistent with the codebase. If something does happen in line 217, we at least would have a log line to help us/customers debug.

Reply:
Have moved the debug message.